scm/driver/github/release.go

```diff
@@ -26,12 +26,14 @@ type release struct {
 }
 
 type releaseInput struct {
-	Title       string `json:"name"`
-	Description string `json:"body"`
-	Tag         string `json:"tag_name"`
-	Commitish   string `json:"target_commitish"`
-	Draft       bool   `json:"draft"`
-	Prerelease  bool   `json:"prerelease"`
+	Title                  string `json:"name"`
+	Description            string `json:"body"`
+	Tag                    string `json:"tag_name"`
+	Commitish              string `json:"target_commitish"`
+	Draft                  bool   `json:"draft"`
+	Prerelease             bool   `json:"prerelease"`
+	DiscussionCategoryName string `json:"discussion_category_name"`
```

Comment:
Probably also want to add omitempty, if it's only a github specific feature?

Reply:
The discussion feature is not enabled by default.